dpnp/dpnp_utils/dpnp_utils_linearalgebra.py

```diff
@@ -925,7 +925,7 @@ def dpnp_matmul(
         # If `order` was not passed as default
         # we need to update it to match the passed `order`.
         if order not in ["k", "K"]:
-            return dpnp.array(result, copy=False, order=order)
+            return dpnp.array(result, copy=None, order=order)
```

Comment:
Is a similar reasoning valid for `dpnp`?

>>> Code using np.array(..., copy=False) can in most cases be changed to np.asarray(...). Older code tended to use np.array like this because it had less overhead than the default np.asarray copy-if-needed behavior. This is no longer true, and np.asarray is the preferred function.

```suggestion
            return dpnp.asarray(result, order=order)
```

Reply:
Yes, it will also work with dpnp. Thank you for the suggestion.